src/PluginsSystem/Tools/Microsoft.Performance.Toolkit.Plugins.Cli/Commands/Pack/PackOptionsValidator.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Performance.Toolkit.Plugins.Cli.Commands.Common;
+using Microsoft.Performance.Toolkit.Plugins.Cli.Console.Verbs;
+using Microsoft.Performance.Toolkit.Plugins.Runtime.Package;
+
+namespace Microsoft.Performance.Toolkit.Plugins.Cli.Commands.Pack
+{
+    internal class PackOptionsValidator
+        : PackGenCommonOptionsValidator<PackOptions, PackArgs>
+    {
+        public PackOptionsValidator(ILogger<PackOptionsValidator> logger)
+            : base(logger)
+        {
+        }
+
+        public override bool TryValidate(PackOptions cliOptions, out PackArgs? validatedAppArgs)
+        {
+            validatedAppArgs = null;
+            if (!TryValidateCommonOptions(cliOptions, out PackGenCommonArgs validatedCommonArgs))
```

Comment:
Instead of having TryValidate be abstract and having each implementation call TryValidateCommonOptions, it might be better to create two methods on `PackGenCommonOptionsValidator`

```cs
public bool TryValidate(TOptions options, out TArgs args)
{
    if (TryValidateCommonOptions(options, out PackGenCommonArgs validatedCommonArgs))
    {
        return TryValidateCore(options, validatedCommonArgs, out args);
    }
    else
    {
        args = null;
        return false;
    }
}

protected abstract bool TryValidateCore(TOptions options, PackGenCommonArgs validatedAppArgs , out TArgs args);
```

This follows the pattern we do in the rest of the SDK